README.md

```diff
@@ -72,9 +72,8 @@ Following urls are accessible after the project is successfully started.
 | [http://localhost/](http://localhost/)           | The application.               |
 | [http://localhost:8081/](http://localhost:8081/) | MySQL management interface.    |
 | [http://localhost:8082/](http://localhost:8082/) | Redis management interface.    |
-| [http://localhost:8083/](http://localhost:8083/) | RabbitMq management interface. |
```

Comment:
It has already been removed. However, the overview dashboard isn't yet a replacement for the whole management interface. A dashboard for queues and possibly exchanges still needs to be added.